validation/validation_runner.py

```diff
@@ -140,13 +132,13 @@ def comparison_job(
     comparison_folder :
     stratification : path to stratification BED data
     """
-
+    batch = get_batch()
     job = batch.new_job(name=f'Compare {sample}')
 
     if dependency is not None:
         job.depends_on(dependency)
 
-    job.image(image_path('happy'))
+    job.image(get_config()['image']['hap.py'])
```

Comment:
To reflect the name in the config:
```suggestion
    job.image(get_config()['image']['happy'])
```